src/multi_vector_simulator/D1_model_components.py

```diff
@@ -414,11 +413,7 @@ def transformer_constant_efficiency_optimize(model, dict_asset, **kwargs):
             inputs = {}
             index = 0
             for bus in dict_asset[INPUT_BUS_NAME]:
-                variable_costs = dict_asset[DISPATCH_PRICE][VALUE][index]
-                inputs[kwargs[OEMOF_BUSSES][bus]] = solph.Flow(
-                    existing=dict_asset[INSTALLED_CAP][VALUE],
-                    variable_costs=variable_costs,
-                )
+                inputs[kwargs[OEMOF_BUSSES][bus]] = solph.Flow()
                 index += 1
             outputs = {
                 kwargs[OEMOF_BUSSES][dict_asset[OUTPUT_BUS_NAME]]: solph.Flow(
```

Comment:
Isnt this part (and the part below) a duplication of the same code that is used for transformers that are fix/to be optimized? You could summarize it in a single processing function